NewForce-at-Mountwest/bangazon-api-gnocci
# Payment Type (Ticket #3)

# Description

Added IsArchived Boolean to Payment Type Model (PaymentTypeController.cs).

Created Controller for Payment Type Resource (PaymentTypeController.cs) and Testing Class / Methods (TestPaymentType.cs) that Validate Operations:
1. GET All
2. GET Single
3. POST
4. PUT
5. DELETE (Soft/Hard)


## Related Ticket(s)
Ticket 3:
https://trello.com/c/4l2BKwUk/3-ticket-3-allow-developers-to-access-payment-type-resource


## Steps to Test

1. git add and git commit whatever you're working on (on your branch)
2. git checkout master, to switch to our local master branch
3. git fetch --all will give you access to my branch
4. git checkout jw-paymenttype to switch to my branch
5. If you don't see the most recent version of my files, try git pull origin jw-paymenttype
6. start BangazonAPI.sln
7. CTRL+R, A to run tests
8. Use SSMS / SQL Viewer to SQL query API ("SELECT * FROM PaymentType")
9. If something is crab-hammered, tell me - don't write any code on my branch
10. Comment on my PR to tell me "great job" or "you fail" and why


# Checklist:
